### PR TITLE
Bump primitive-types to 0.13.1

### PR DESCRIPTION
**Pull request by pgherveou**

### CODE REVIEW

File: primitive-types/Cargo.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ rust-version = "1.60.0"
 fixed-hash = { version = "0.8", path = "../fixed-hash", default-features = false }
 uint = { version = "0.10.0", path = "../uint", default-features = false }
 impl-serde = { version = "0.5.0", path = "impls/serde", default-features = false, optional = true }
-impl-codec = { version = "0.6.0", path = "impls/codec", default-features = false, optional = true }
+impl-codec = { version = "0.6.1", path = "impls/codec", default-features = false, optional = true }
```
Comment (pgherveou) [inline primitive-types/Cargo.toml:16]:
```suggestion
impl-codec = { version = "0.6", path = "impls/codec", default-features = false, optional = true }
```

File: primitive-types/Cargo.toml [hunk 2]
```diff
@@ -11,10 +11,10 @@ rust-version = "1.60.0"
 
 [dependencies]
 fixed-hash = { version = "0.8", path = "../fixed-hash", default-features = false }
-uint = { version = "0.10.0", path = "../uint", default-features = false }
-impl-serde = { version = "0.5.0", path = "impls/serde", default-features = false, optional = true }
-impl-codec = { version = "0.6.0", path = "impls/codec", default-features = false, optional = true }
```
Comment (pgherveou) [inline primitive-types/Cargo.toml:16]: `impl-codec@0.6.0` was wrong since the macro generates code that uses the latest breaking changes

**Comment by pgherveou** [comment]
@ordian  I believe we should yank 0.13 and release 0.13.1 instead

**Comment by ordian** [comment]
> @ordian I believe we should yank 0.13 and release 0.13.1 instead [This breaks ](https://github.com/paritytech/polkadot-sdk/pull/5673) does not build with newest versions because of that

Sorry I don't understand the changes in this PR. `version = "0.6.0"` means `version = "ˆ0.6.0"`, which is any compatible version with `0.6.0`, which include `0.6.1`.
https://crates.io/crates/primitive-types/0.13.0/dependencies

Could you elaborate what exactly was wrong?

**Pull request closed without merging**